scripts/install-atom.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+ATOM_CHANNEL="${ATOM_CHANNEL:=stable}"
+
+curl -s -L "https://atom.io/download/deb?channel=${ATOM_CHANNEL}" \
+  -H 'Accept: application/octet-stream' \
+  -o "atom-amd64.deb"
+dpkg-deb -x atom-amd64.deb "${HOME}/atom"
+
+if [ "${ATOM_CHANNEL}" = "stable" ]; then
+  export ATOM_SCRIPT_NAME="atom"
+  export APM_SCRIPT_NAME="apm"
+else
+  export ATOM_SCRIPT_NAME="atom-${ATOM_CHANNEL}"
+  export APM_SCRIPT_NAME="apm-${ATOM_CHANNEL}"
+fi
+
+export ATOM_SCRIPT_PATH="${HOME}/atom/usr/bin/${ATOM_SCRIPT_NAME}"
+export APM_SCRIPT_PATH="${HOME}/atom/usr/bin/${APM_SCRIPT_NAME}"
+export NPM_SCRIPT_PATH="${HOME}/atom/usr/share/${ATOM_SCRIPT_NAME}/resources/app/apm/node_modules/.bin/npm"
+
+echo "Using Atom version:"
+"${ATOM_SCRIPT_PATH}" -v
+echo "Using APM version:"
+"${APM_SCRIPT_PATH}" -v
+
+echo "Downloading package dependencies..."
+"${APM_SCRIPT_PATH}" clean
+
+export NPM_SCRIPT_PATH="npm"
+"${APM_SCRIPT_PATH}" install --production
+
+# Use the system NPM to install the devDependencies
+echo "Using Node version:"
+node --version
+echo "Using NPM version:"
+npm --version
```

Comment:
I wonder if we could use https://github.com/wooorm/atom-travis instead. Not a big fan of including so much boilerplate.